tests/timeseries/test_fill_missing_timestamps.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import pandas as pd
+from janitor.timeseries import fill_missing_timestamps
+
+
+@pytest.mark.timeseries
+def test_datatypes_check():
+    with pytest.raises(TypeError):
+        assert fill_missing_timestamps(
+            4, '1T', None, None, None
+        )
+        assert fill_missing_timestamps(
+            pd.DataFrame([]), 2, None, None, None
+        )
+        assert fill_missing_timestamps(
+            pd.DataFrame([]), '1B', 1, None, None
+        )
+        assert fill_missing_timestamps(
+            pd.DataFrame([]), '1B', 'DateTime', '2020-01-01', None
+        )
+        assert fill_missing_timestamps(
+            pd.DataFrame([]), '1B', 'DateTime', pd.Timestamp(2020, 1, 1), '2020-02-01'
+        )
+
```

Comment:
Apart from the error-checking test above, I think it may help to provide a test that is small enough for others to read and execute inside a test. Could you add that in please?

Reply:
Perhaps something like this is what you have in mind ? 

```python
@pytest.mark.timeseries
def test_fill_missing_timestamps():
    assert fill_missing_timestamps(
        pd.DataFrame([]), '1B', 'DateTime', pd.Timestamp(2020, 1, 1), pd.Timestamp(2020, 2, 1)
    )
```

Reply:
Almost! The assert isn’t necessary, the return value of `fill_missing_timestamps` is not the thing we need tested, but rather, _properties_ of the return value. For example, an appropriate assertion here is to assert that the timestamps that are added are correct.